microsite/data/plugins/docker-tags.yaml

```
@@ -0,0 +1,10 @@
+---
+title: Docker tags
+author: Workm8
+authorUrl: 'https://github.com/work-m8/.github'
+category: Services
+description: 'Creates a list of docker tags based on hub.docker.com'
+documentation: 'https://raw.githubusercontent.com/Work-m8/backstage-docker-plugin/main/README.md'
```

```suggestion
documentation: 'https://github.com/Work-m8/backstage-docker-plugin/blob/main/README.md'
```

This get's you the raw Markdown, you probably want to use this 👆